Tasks/XamarinAndroid/task.json

```diff
@@ -99,7 +99,7 @@
             "required": false,
             "groupName": "msbuildOptions",
             "defaultValue": "15.0",
-            "helpMarkDown": "If the preferred version cannot be found, the latest version found will be used instead. On a Mac xbuild (Mono) will be used.",
+            "helpMarkDown": "If the preferred version cannot be found, the latest version found will be used instead. On macOS xbuild (Mono) or MSBuild (Visual Studio for Mac) will be used.",
```

Comment:
P999: comma after "macOS":
"On macOS, xbuild (Mono) or MSBuild (Visual Studio for Mac) will be used."